src/mono/mono/mini/simd-intrinsics-netcore.c

```diff
@@ -261,12 +261,39 @@ get_vector_t_elem_type (MonoType *vector_type)
 	klass = mono_class_from_mono_type_internal (vector_type);
 	g_assert (
 		!strcmp (m_class_get_name (klass), "Vector`1") || 
+		!strcmp (m_class_get_name (klass), "Vector64`1") ||
 		!strcmp (m_class_get_name (klass), "Vector128`1") || 
 		!strcmp (m_class_get_name (klass), "Vector256`1"));
 	etype = mono_class_get_context (klass)->class_inst->type_argv [0];
 	return etype;
 }
 
+static MonoInst *
+emit_arch_vector64_create_multi (MonoCompile *cfg, MonoMethodSignature *fsig, MonoClass *klass, MonoType *etype, MonoInst **args)
+{
+#if defined(TARGET_AMD64)
+	MonoInst *ins, *load;
```

Comment:
The commit message says Arm64.

Reply:
Would be nice to add a helper function to unify this with the emit_arch_vector128_create_multi () function, they are mostly the same except the 16/8.